config/values.yml

```diff
@@ -1,8 +1,5 @@
-#@data/values
+#@data/values-schema
 ---
 namespace: secretgen-controller
 create_namespace: true
-
-push_images: false
-image_repo: docker.io/k14s/secretgen-controller-test
-secretgen_controller_version: "develop"
+secretgen_controller_version: "develop"
```

Comment:
should this be under `dev` since this is only used for building afaik, but hmm it also gets used in the annotation...

Reply:
Right, it seems to me like the only way to not have it as a value is to hardcode it in.